pyaerocom/scripts/cams2_83/cli.py

```diff
@@ -286,7 +351,12 @@ def main(
         help="Will only make and print the config without running the evaluation",
     ),
     pool: int = typer.Option(
-        1, "--pool", "-p", min=1, max=cpu_count(), help="CPUs for reading OBS"
+        1,
+        "--pool",
+        "-p",
+        min=1,
+        max=cpu_count(),
```

Comment:
Don't set cpu_count() as maximum if you don't know if this is right.

Some tasks are IO bound, so a maximum of 4 is often too much in this case. In theory, we could have tasks which connect to external resources, so that we might want to use much more than the cpus available. (A process based webserver setup used to be 2-4 times the number of cores.)

We don't need to add a maximum here.